HousingSearchApi/V1/Boundary/Requests/GetTenureListRequest.cs

```diff
@@ -6,5 +6,11 @@ public class GetTenureListRequest : HousingSearchRequest
     {
         [FromQuery(Name = "uprn")]
         public string Uprn { get; set; }
+
+        [FromQuery(Name = "bookingStatus")]
+        public string BookingStatus { get; set; }
+
+        [FromQuery(Name = "isTemporaryAccommodation")]
+        public string IsTemporaryAccommodation { get; set; }
```

Comment:
Should this be bool to be inline with other boundary objects?

Reply:
I would of preferred to use bool as well however the implementation for how the filters generate the query only work with strings. It would be a large re-engineering task to change the query builder to work with other types. The asset search works in the same way (here are examples for the asset requests: [Example 1](https://github.com/LBHackney-IT/housing-search-api/blob/master/HousingSearchApi/V1/Boundary/Requests/GetAllAssetListRequest.cs#L31-L32) &  [Example 2](https://github.com/LBHackney-IT/housing-search-api/blob/master/HousingSearchApi/V1/Boundary/Requests/GetAssetListRequest.cs#L13-L14) which have filterable booleans as strings) which was the approach we agreed on for this ticket. Maybe in the future we can change it but for now its best to keep it as is.